### PR TITLE
Run credo as part of the ci build

### LINKED ISSUE #24: CI task should also check credo warnings/errors
We should go through some rules/warnings in credo that we deem sufficient to fail the build.
Leaving a stray `IO.inspect` would be the first candidate.

As such, we should also create a separate mix task or alias to run during CI builds.

### PR DESCRIPTION
**Pull request by felipesere**
Running `mix credo --strict` throws no warnings at the moment.
To see what CI would do at the moment run `MIX_ENV=test mix ci`
Rules are up for debate...

Closes #24

### CODE REVIEW

File: lib/advisor/web/questionnaire_proposal.ex [hunk 1]
```diff
@@ -27,6 +28,9 @@ defmodule Advisor.Web.QuestionnaireProposal do
     end
   end
 
+  def truthish({_key, "true"}), do: true
```
Comment (RabeaGleissner) [inline lib/advisor/web/questionnaire_proposal.ex]: Is this a word? Maybe `truthy`?

File: mix.exs [hunk 2]
```diff
@@ -50,6 +50,8 @@ defmodule Advisor.Mixfile do
   defp aliases do
     ["ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
      "ecto.reset": ["ecto.drop", "ecto.setup"],
-     "test": ["ecto.create --quiet", "ecto.migrate", "test"]]
+     "test": ["ecto.create --quiet", "ecto.migrate", "test"],
+     "ci": ["test", "credo --ignore readability"]
```
Comment (RabeaGleissner) [inline mix.exs]: does this mean that the CI does not check for readability?

Comment (felipesere) [inline mix.exs]: excellent catch. Initially I thought if leaving it off, but by now I have fixed all issues concerning readability. I'll fix it tonight.

File: lib/advisor/web/questionnaire_proposal.ex (hunk 1 above)
Comment (felipesere) [inline lib/advisor/web/questionnaire_proposal.ex]: Will do.

**Pull request merged**